src/utils/fileDialog.ts

```diff
@@ -0,0 +1,39 @@
+interface IFileDialogOptions {
+  onChange: (files?: FileList) => void;
+  multiple?: boolean;
+  accept?: string;
+}
+
+const defaultDialogOptions: IFileDialogOptions = {
+  onChange: () => {},
+  multiple: false,
+  accept: '*',
+};
+
+function createInputEl(): HTMLInputElement {
+  const input = document.createElement('input');
+  input.type = 'file';
+  input.hidden = true;
+  input.className = 'hidden';
+  document.body.appendChild(input);
+  return input;
+}
+
+export function openFileDialog(opts: IFileDialogOptions = defaultDialogOptions): Promise<void> {
```

Comment:
Few more suggestions:
1. I still think this util could be shrinked and placed in the global util files. Heres how:
2. The function in my opinion should not return `Promise<void>` but `Promise<FileList>`. So It should be used in a following way:

```
try {
  const files = await selectFiles();
} catch {
  // User clicked cancel or something else... we do nothing?
}
```
3. With this approach there would be no need to have separate definition for the options and const for default options. It can go this way:
```
export function selectFiles(options: { multiple?: boolean; accept?: string } = { multiple: false, accept: '*' }) {
```
4. Also I don't see much value in having separate util `createInputEl`.